recipes/mpmcqueue/all/test_package/CMakeLists.txt

```diff
@@ -0,0 +1,12 @@
+cmake_minimum_required(VERSION 3.1)
+project(test_package)
+
+include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
+conan_basic_setup(TARGETS)
+
+find_package(MPMCQueue REQUIRED CONFIG)
+find_package(Threads REQUIRED)
```

Comment:
Should this be a system lib?

Reply:
No, see https://github.com/conan-io/conan-center-index/pull/5955#discussion_r654015050

Reply:
Ahhhhhh it's for your test thanks for confirming